src/commonMain/kotlin/exchange.dydx.abacus/calculator/TradeInputCalculator.kt

```diff
@@ -1748,19 +1730,25 @@ internal class TradeInputCalculator(
         }
     }
 
-    private fun marketOrderBestPrice(marketOrder: Map<String, Any>): Double? {
-        parser.asNativeList(marketOrder["orderbook"])?.let { orderbook ->
-            if (orderbook.isNotEmpty()) {
-                parser.asNativeMap(orderbook.firstOrNull())?.let { firstLine ->
-                    parser.asDouble(firstLine["price"])?.let { bestPrice ->
-                        if (bestPrice != Numeric.double.ZERO) {
-                            return bestPrice
-                        }
-                    }
+    private fun marketOrderbookMidPrice(market: Map<String, Any>?): Double? {
+        return parser.asNativeMap(market?.get("orderbook_consolidated"))?.let { orderbook ->
+            parser.asDouble(parser.value(orderbook, "asks.0.price"))?.let { firstAskPrice ->
+                parser.asDouble(parser.value(orderbook, "bids.0.price"))?.let { firstBidPrice ->
+                    (firstAskPrice + firstBidPrice) / 2.0
                 }
             }
         }
-        return null
+    }
+
+    private fun marketOrderSlippageFromMidPrice(worstPrice: Double?, midMarketPrice: Double?): Double? {
+        return if (worstPrice != null && midMarketPrice != null && midMarketPrice > Numeric.double.ZERO) {
+            Rounder.round(
```

Comment:
can i use `Rounder.round` here? what's the difference between this and `Rounder.quickRound`? @prashanDYDX